candi/bashible/bundles/ubuntu-lts/all/012_apt_timeout.sh.tpl

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ -f /etc/apt/apt.conf.d/99timeout ]]; then
+  return 0
+fi
+
+echo 'Acquire::http::Timeout "120";' > /etc/apt/apt.conf.d/99timeout
```

Comment:
What to do with apt in previous steps ?

Reply:
The probability of blocking is quite low when the node is first set up. it is the minute-by-minute startup that causes the biggest problem.